utils/build/docker/nodejs/express/app.js

```diff
@@ -237,6 +237,14 @@ app.get('/kafka/consume', (req, res) => {
     })
 })
 
+app.get('/log/library', (req, res) => {
```

Comment:
Can we add a test for this new endpoint here: https://github.com/DataDog/system-tests/blob/main/tests/parametric/test_parametric_endpoints.py#L4. This will allow us to report to the feature parity dashboard on which endpoints are not implemented in different languages.

Also to make this endpoint more useful can we add the following required arguments:
 - log level (warn/error/info/debug/...), we can use an enum here and ensure the casing is always consistent.
 - message (right now it's optional)